README.md

```diff
@@ -82,9 +82,12 @@ Should start with `is`, `has`, `will` or `should`. Like `isValid` or `hasValues`
 #### Always Affirmative
 Avoid negations. _“Don’t ever not avoid negative logic”_. Prefer `isShown` over `isHidden` or `isEnabled` over `isDisabled`. Do not use names like `notEditable`.
 
+#### TypeScript Generics
+If a Class or Interfaces uses only one generic and the usages of the generic is obvious, then naming that generic `T` is sufficient. If a Class or Interface has more generics or when the usage is not obvious, you should use a more descriptive name. The same naming rules as for classes will apply then. 
```

Comment:
How about prefixing generic names with a `T`? E.g. `<TKey, TValue>`

I'm not a fan, it's similar to how we don't do that with Interfaces (anymore) (right?).

Reply:
I don't like prefixing. And there is also no need for it.

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generic-type-parameters

Some have different opinions, just throwing it out here :) (they also do the interface prefix :P)